apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -551,9 +551,9 @@ export const ConversationComponent = ({ item: conversation, level }: Props) => {
         <button
           className={classNames(
             'group flex size-full cursor-pointer items-center gap-2 transition-colors duration-200',
-            messageIsStreaming && 'disabled:cursor-not-allowed',
-            isDeleting && 'pr-12',
-            !messageIsStreaming && !isDeleting && 'group-hover:pr-6',
+            messageIsStreaming
+              ? 'disabled:cursor-not-allowed'
+              : 'group-hover:pr-6',
```

Comment:
`disabled:cursor-not-allowed [&:not(:disabled)]:group-hover:pr-6` can be added as static on previous line